### DISCUSSION TITLE
Can I specify dynamic parameter values .csv files at scenario level

### DISCUSSION THREAD

**Discussion by harimadusumilli** [getgauge/gauge#1949] [General]
Can we use CSV file to pass dynamic parameters at the scenario level ? In the Gauge Documentation, I see that there are examples where the csv files can be passed at the step level. I would like to know if we can pass the .csv file at the scenario level, so that the whole scenario can get repeatedly executed for different sets of data as provided in the .csv file.

I was unable to find an example, so posted in the group.

**Comment by sriv** [comment]
Yes. Something like this should work:

```
## Vowel counts from file

table:vowels.csv

* The word <word> has <count> vowels.
```

Note that you still need to set `allow_scenario_datatable=true`.
